app/utils/DocumentUpload.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+class DocumentUpload {
+    static initDocuments(formdata) {
+        if (!formdata.documents) {
+            formdata.documents = {};
+        }
+        return formdata;
+    }
+
+    static addDocument(uploadedFile, uploads = []) {
+        if (uploadedFile) {
+            uploads.push(uploadedFile);
+        }
+        return uploads;
+    }
+
+    static removeDocument(index, uploads) {
```

Comment:
Will you be adding unit test cases for this function?

Reply:
Thanks...I do have more unit tests for that class coming an a later PR and those do include some for `removeDocument()` 😄